easybuild/framework/easyblock.py

```diff
@@ -214,6 +214,18 @@ def _init_log(self):
 
         self.log = fancylogger.getLogger(name=self.__class__.__name__, fname=False)
 
+        # deprecate self.log.error in easyblocks, self.log.fatal should be used instead
+        def error(msg, *args, **kwargs):
```

Comment:
no, a call to `self.error(msg, args)` inside an easyblock will result in a call  `EasyBlock.error(self, msg, args)` because it's a class method.